.travis.swagger.sh

```diff
@@ -17,10 +17,14 @@
 #
 
 # generate the docs
-mvn clean compile -Pdocgen -DskipTests -Dcheckstyle.skip
+mvn clean compile -Pdocgen -DskipTests -Dcheckstyle.skip -Dlicense.skip
```

Comment:
This is run in post success 'travis phase' so there is no need for this license check because it was already done before.

So it should speed up the doc generation process.